### PR TITLE
Merging SLE 12 SP2 into master

### PR DESCRIPTION
**Pull request by gilsonsouza**

### CODE REVIEW

**Comment by teclator** [comment]
From my POV it should be merged in https://github.com/yast/yast-installation/tree/SLE-12-SP2-CASP and then from it into master.

@imobachgs could you check.

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   - docker
 
 before_install:
-  - docker build -t yast-installation-image .
+    - docker build -t yast-installation-image .
```
Comment (imobachgs) [inline .travis.yml]: Is this change intentional?
